### ISSUE TITLE
Needs support for Other Bookmarks folder

### ISSUE THREAD

**Issue by Gitoffthelawn** [heftig/sort-bookmarks#12]
Ah, I had such high hopes!

Excellent extension.  Thank you!  But it sorted the entire `Other Bookmarks` folder, and there is no way to specify a custom sort (or no sort) for that critical folder.

Fortunately, I made a backup first, but I realized that doesn't backup the thousands of favicons.

Can you add support for the `Other Bookmarks` folder?

**Comment by heftig** [comment]
Yeah, Firefox doesn't show custom menu items for special folders. But you can still sort it from the menu of one of its bookmarks.

**Comment by Gitoffthelawn** [comment]
Thank you so very much for your quick reply!

You're right, you can still sort it from the menu of one of its bookmarks.  That's a good workaround!  I wish I knew that before I messed everything up!  Live and learn.

Are you willing to file a bugzilla issue regarding this problem so maybe some day, in 20 years, Mozilla will fix it?  I think they tend to respond better to the developers of extensions that are affected than users of those extensions.
